spec/text-editor-presenter-spec.coffee

```diff
@@ -1893,20 +1848,18 @@ describe "TextEditorPresenter", ->
 
             getLineNumberGutterState(presenter).content.lineNumbers[key]
 
-          it "contains states for line numbers that are visible on screen, plus and minus the overdraw margin", ->
+          it "contains states for line numbers that are visible on screen", ->
```

Comment:
I'm worried about the repaint behavior of the gutter if it doesn't overdraw and isn't tiled. Who knows what's changed in Chromium's graphics implementation, but previously when I didn't overdraw the red paint rectangle always covered the entire gutter when scrolling.